Azure/sonic-buildimage
# Convert radv docker to buster

**- Why I did it**
This is an enhancement.

**- How I did it**
Code changes

**- How to verify it**
Build radv docker using `BLDENV=buster make -f Makefile.work  target/docker-router-advertiser.gz`
Check that /etc/apt/sources.list in radv docker references buster.

**- Description for the changelog**
Changes to convert radv (router-advertiser) docker to buster

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>


**- A picture of a cute animal (not mandatory but encouraged)**
